content/docs/faq.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Frequently Asked Questions
+---
+
+## General
+
+### Why do you say "Tina is not a CMS?"
+
+Tina is a suite of packages that enable developers to build highly customized content management systems. A conventional CMS provides a fully-realized editing interface, requiring you to create an edit content within its bounds. Tina takes an "inside-out" approach, helping developers craft their own CMS according to their specific content needs.
```

Comment:
"Enable to build custom CMS" does not tell the whole story and explain the new paradigm. 
How to put that you'll not build a traditional backend CMS, but adding editing capabilities on your website front-end? 
Is front-end content management system a thing?

Reply:
_Warning: This is kind of rambly, I'm tired_

Front-end CMS isn't really a thing but I like where you're going.

Tina is not a CMS, and it's not just a toolkit for building CMSs. It's a toolkit for making your website it's own CMS. 

* Traditional Paradigm: A CMS is an application you use to build a website (i.e. wordpress, drupal, etc.) You build your website on top of the CMS.
* Headless Paradigm: A CMS is an application that let's you edit your content. It lives independent of your website. Your website's build process loads data from the CMS to create the web pages
* The Tina Paradigm: Your website is it's own CMS. You use Tina to build content editing capabilities directly into your websites interface. 

A curt-er way to think about it might be this:

- Your website is INSIDE the CMS
- Your website is BESIDE the CMS
- Your website IS the CMS

Reply:
The thing about the Tina Paradigm is it's actually working at a higher level then either of the other two. 

- You could build a traditional CMS using Tina
- You can use Tina with Headless CMSs

Reply:
When you coin a new term like "Content Platform", or "Jamstack", or "Uphill Charts" or "Imbox", even if the term will always be debatable, the point is you're introducing a real change or feature and bringing something new to the game. 

This is marketing territory, but I feel Tina deserves to coin its own term to convey the paradigm you're describing. 
CMS builder/Edition Layer I really don't know how to call it.